### ISSUE TITLE
Check for updates (Windows) failing

### ISSUE THREAD

**Issue by JohnnyOpcode** [thiagoralves/OpenPLC_Editor#53]
The current version downloaded from the website detects a new version, but fails to download (?) and install (?) in Windows (10).

ANy ideas where to patch?

**Comment by thiagoralves** [comment]
It is working for me. What is the error message you're getting?

**Comment by JohnnyOpcode** [comment]
It hangs shortly after the download is my best interpretation.
![image](https://user-images.githubusercontent.com/1428214/183094151-23660058-f017-416b-9fc5-5405e759bbd7.png)

**Comment by thiagoralves** [comment]
Check your internet connection. There could be something blocking access to
github servers. Make sure there aren't any firewall or antivirus blocking
OpenPLC Editor traffic.

**Comment by JohnnyOpcode** [comment]
Do you mean I need to add a firewall rule (manually) for the application under Windows 10? I am browsing GitHub without issue as can be seen here.

**Comment by thiagoralves** [comment]
No add a firewall rule, but make sure there isn't any rule *blocking* it.
Firewall rules are per application, so your browser could browse github
normally, but it could be possible that there is a rule blocking OpenPLC
Editor external communications

**Comment by JohnnyOpcode** [comment]
C:\Users\DEV\OpenPLC_Editor>if exist .\new_editor\ rmdir /s /Q new_editor

C:\Users\DEV\OpenPLC_Editor>if exist .\OpenPLC_Editor\ rmdir /s /Q OpenPLC_Editor

C:\Users\DEV\OpenPLC_Editor>git clone https://github.com/thiagoralves/OpenPLC_Editor
Cloning into 'OpenPLC_Editor'...
fatal: unable to access 'https://github.com/thiagoralves/OpenPLC_Editor/': error setting certificate verify locations:  CAfile: C:\Program Files\Git\mingw64\ssl\certs\ca-bundle.crt CApath: none

C:\Users\DEV\OpenPLC_Editor>if exist .\OpenPLC_Editor\editor\ (
move .\OpenPLC_Editor\editor .\new_editor
 copy /y .\OpenPLC_Editor\revision .\
 rmdir /s /Q OpenPLC_Editor
 echo "Update applied successfully"
)  else (echo "Error cloning from repository!" )
"Error cloning from repository!"

C:\Users\DEV\OpenPLC_Editor>

**Comment by thiagoralves** [comment]
The updater is trying to use Git from your machine but apparently it is
broken (missing certificates). OpenPLC Editor includes its own Git. Try
reinstalling Git on your machine, or removing it altogether, so that the
updater will use the bundled Git.

**Comment by JohnnyOpcode** [comment]
I have no Git installed or on my path. I do have GitHub Desktop however.

This directory does not exist on my PC nor the .crt file as well.
C:\Program Files\Git\mingw64\ssl\certs\ca-bundle.crt

I will keep debugging the problem..

**Comment by LeaderXinjianSun** [comment]
Hi, Can I change defult url 'https://github.com/thiagoralves/OpenPLC_Editor.git' to my forked Repository 'https://gitee.com/sunxinjian/OpenPLC_Editor.git'? It may be faster than clone from github directly.

**Comment by thiagoralves** [comment]
Yes, just change the URL from the update script at the root of your OpenPLC Editor installation
